telegram_bot/telegram_bot.py

```diff
@@ -431,6 +430,12 @@ def handle_user_response(chat_id, response):
         # prompt_to_repeat_group(chat_id=chat_id)
         return
     
+    if answer_from_user and response is not None:
+        q = session.stage.prompt
+        show_answer = f'{q}: {session.humanize_answer(response)}'
+        bot.edit_message_text(chat_id=chat_id, message_id=session.last_question_message.id, text=show_answer, reply_markup=None)
```

Comment:
I think it would be nicer for this to be done in a method of `Session` instead of accessing from here `session.last_question_message` (it's more in line with the rest of the implementation)